Sources/BraintreeCore/Analytics/FPTIBatchData.swift

```diff
@@ -64,10 +67,12 @@ struct FPTIBatchData: Codable {
             endTime: Int? = nil,
             errorDescription: String? = nil,
             eventName: String,
+            merchantExperiment: String? = nil,
             isConfigFromCache: Bool? = nil,
             isVaultRequest: Bool? = nil,
             linkType: String? = nil,
             payPalContextID: String? = nil,
+            paymentMethodsDisplayed: [String?] = [],
```

Comment:
Super nit m comes before p so we will want to bump this up one and same in a few places bellow